src/gui/mainwindow.cpp

```diff
@@ -213,8 +225,19 @@ MainWindow::MainWindow(IGUIApplication *app, WindowState initialState)
         UIThemeManager::instance()->getIcon(u"folder-remote"_qs),
 #endif
         tr("Transfers"));
-
-    connect(m_searchFilter, &LineEdit::textChanged, m_transferListWidget, &TransferListWidget::applyNameFilter);
+    // Filter types
```

Comment:
There is no benefit from this comment, IMO. I would delete it along with the next empty line.